.gitignore

```diff
@@ -112,5 +112,14 @@ dmypy.json
 ### Virtual environment ###
 env/
 
+### vscode settings ###
+.vscode/
+app_output.html
+not_an_init_.py
+
 ### Configuration files ###
 .env
+
+#  Selenium Driver and logs
+geckodriver.exe
```

Comment:
While this correctly in the `.gitignore` file it is still in source control.

To remove the file from source control only please run ` git rm --cached .\geckodriver.exe`.